ts/Core/Pointer.ts

```diff
@@ -1436,7 +1436,8 @@ class Pointer {
 
         // On touch devices, only proceed to trigger click if a handler is
         // defined
-        if (hasZoom && self.initiated && !fireClickEvent && e.cancelable) {
+        if (hasZoom && self.initiated && !fireClickEvent &&
+            (typeof e.cancelable !== 'boolean' || e.cancelable)) {
```

Comment:
Just nitpicking, but you might be able to get away with `e.cancelable !== false` and save a few bytes of filesize.